src/services/config.service.ts

```diff
@@ -7,6 +7,25 @@ import { PersonalShelf } from '#src/enum/PersonalShelf';
  * Set config setup changes in both config.services.ts and config.d.ts
  * */
 
+/**
+ * We check here that we:
+ * 1. Added favorites_list / continue_watching_list feature
+ * 2. Included a corresponding element (with favorites or continue_watching type) in the content array
+ */
+const checkAdditionalFeatures = (content: Content[], playlistId: string | undefined | null, type: PersonalShelf) => {
```

Comment:
This check seems a bit heavy handed and I think it will cause us more trouble than good. We've talked before about potentially having favorites or continue watching show up as libraries instead of in the content. It's possible that other use cases will arise where we want to set the playlist ID's but not have a content item.

Reply:
I have answered in the previous thread 😄 Do you think that we should add content items with `favorites` and `continue_watching` on our own if the customer only set the playlist id?

Reply:
See above, but I think it's fine to just allow it